evaluator.go

```diff
@@ -104,26 +130,132 @@ func (state *State) SetDefaultCluster(name string) {
 // PrimeCache traverses over the entire state to expand all values and store
 // them in the state's cache. Subsequent queries will be able to use the cache
 // immediately, rather than having to build it up incrementally.
-//
+
```

Comment:
I think this will exclude the top half from the generated documentation, so empty `//` is actually required.

https://godoc.org/github.com/square/grange

I'm pretty sure there's an easy way to generate this locally to verify...